common/djangoapps/util/password_policy_validators.py

```diff
@@ -42,12 +44,12 @@ def validate_password_length(value):
     message = _("Invalid Length ({0})")
     code = "length"
 
-    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
-    max_length = getattr(settings, 'PASSWORD_MAX_LENGTH', None)
+    min_length = PASSWORD_MIN_LENGTH
```

Comment:
I will go with the second one because other validations also lies there. What is the right way to define those settings. Do I need to create PR or we need to ask devops.